inc/class-wp-cli-command.php

```diff
@@ -18,24 +18,23 @@ public function output( $args, $args_assoc ) {
 		}
 
 		$urls = ! empty( $args[0] ) ? [ $args[0] ] : get_site_urls( $args_assoc['config'] );
+		$urls = array_unique( $urls );
 
-		foreach ( $urls as &$url ) {
-			$url = get_url_contents( $url, $args_assoc['config'] );
-			if ( is_wp_error( $url ) ) {
-				WP_CLI::warning( $url->get_error_message() );
-				continue;
-			}
+		$contents = array_map( __NAMESPACE__ . '\\get_url_contents', $urls );
 
-			$url = replace_urls( $url, $args_assoc['config'] );
-		}
+		$contents = array_map( __NAMESPACE__ . '\\replace_urls', $contents );
+
+		array_map( function( $content, $url ) use ( $args_assoc ) {
```

Comment:
I'm not sure why this was added? This CLI command is to just show the HTML output, not save it too.